src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplateInfoSection.tsx

```diff
@@ -0,0 +1,147 @@
+  const interfaces = getTemplateInterfaces(template);
+  const disks = getDisks(vm);
+  const isDefaultTemplate = isDefaultVariantTemplate(template);
+  const [isTemplateInfoExpanded, setIsTemplateInfoExpanded] = useState(true);
+
+  return (
+    <ExpandableSection
+      isExpanded={isTemplateInfoExpanded}
+      isIndented
+      onToggle={setIsTemplateInfoExpanded}
+      toggleText={t('Template info')}
+    >
+      <DescriptionList>
+        <DescriptionListGroup>
+          <DescriptionListTerm>{t('Operating system')}</DescriptionListTerm>
+          <DescriptionListDescription>{displayName}</DescriptionListDescription>
+        </DescriptionListGroup>
+        <DescriptionListGroup>
+          <DescriptionListTerm>{t('Workload type')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            {WORKLOADS_LABELS[workload] ?? t('Other')} {isDefaultTemplate && t('(default)')}
+          </DescriptionListDescription>
+        </DescriptionListGroup>
+        <DescriptionListGroup>
+          <DescriptionListTerm>{t('Description')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            {<TemplateExpandableDescription description={description} />}
+          </DescriptionListDescription>
+        </DescriptionListGroup>
+        <DescriptionListGroup>
+          <DescriptionListTerm>{t('Documentation')}</DescriptionListTerm>
+          <DescriptionListDescription>
+            {documentationUrl ? (
+              <Button
+                icon={<ExternalLinkSquareAltIcon />}
+                iconPosition="right"
+                isInline
+                isSmall
+                variant="link"
+              >
+                <a href={documentationUrl} rel="noopener noreferrer" target="_blank">
+                  {t('Refer to documentation')}
+                </a>
+              </Button>
+            ) : (
+              notAvailable
+            )}
+          </DescriptionListDescription>
+        </DescriptionListGroup>
+        <AdditionalResources template={template} />
+        <VirtualMachineDescriptionItem
+          bodyContent={
+            <CPUDescription
+              cpu={getCPU(vm)}
+              helperTextResource={CpuMemHelperTextResources.FutureVM}
+            />
+          }
+          onEditClick={() =>
+            createModal(({ isOpen, onClose }) => (
+              <CPUMemoryModal
+                isOpen={isOpen}
+                onClose={onClose}
+                onSubmit={updateVMCPUMemory(vmNamespace, setVM, setVM)}
+                templateNamespace={template?.metadata?.namespace}
+                vm={vm}
+              />
+            ))
+          }
+          descriptionData={<CPUMemory fetchVMI={false} vm={vm} />}
+          descriptionHeader={t('CPU | Memory')}
+          isEdit
+          isPopover
+        />
+        <DescriptionListGroup>
+          <DescriptionListTerm>
+            {t('Network interfaces')}
+            {` (${networks?.length})`}
```

Comment:
can be combined to one

Reply:
why resolved and left like that?
{t('Network interfaces {{length}}, {length: networks?.length})}

Reply:
also u can add plurals so it can be network inerface/s

Reply:
look on example on line 138